charts/member-agent/templates/deployment.yaml

```diff
@@ -105,6 +113,11 @@ spec:
       volumes:
       - name: provider-token
```

Comment:
around this with `{{- if not .Values.useCAAuth }}` and `{{- end }}` after line 115